interfaces/Portal/src/assets/i18n/en.json

```diff
@@ -454,7 +454,9 @@
           },
           "multiple-choice-hidden-options": "Above list only shows a status if assigned to at least one Person Affected"
         },
+        "filter-label": "Select filter:",
         "filter-placeholder": "Type value...",
```

Comment:
This `filter-placeholder` can also be moved to the `filter-row`-related group, I think.